### PR TITLE
Automated recurring emails 202

### PR DESCRIPTION
**Pull request by bledsoef**
Created a script that when executed will check to see if there are any events the next day and if there are, send an email to all of the recipients of each of the events.

Fixes issue #202

### CODE REVIEW

File: app/logic/events.py [hunk 1]
```diff
@@ -318,3 +319,10 @@ def preprocessEventData(eventData):
         eventData['facilitators'] = list(User.select().join(EventFacilitator).where(EventFacilitator.event == event))
 
     return eventData
+
+
+def getTomorrowsEvents():
+    """Grabs each event that occurs tomorrow"""
+    tomorrowDate = date.today() + timedelta(days=1)
+    events = list(Event.select().where(Event.startDate==tomorrowDate))
```
Comment (zachneill) [inline app/logic/events.py:327]: See if you can change list() to the [ ] notation. It actually calculates faster and is easier to read.

Comment (BrianRamsay) [reply app/logic/events.py:327]: WRONG! Actually, I'm not sure, but when I try using `[]` around a select I end up with a list with a select object in it, not a list of the results.

File: app/scripts/sendEventReminderEmails.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        counter = 0
+        currentTerm = Term.select(Term.id).where(Term.isCurrentTerm==1)
+        template = EmailTemplate.get(purpose = "Reminder")
+        templateSubject = template.subject
+        templateBody = template.body
+        for event in events:
+            programId = event.singleProgram
+            emailData = {"eventID":event.id,  # creates the email data
+                            "programID":programId,
+                            "term":currentTerm,
```
Comment (BrianRamsay) [inline app/scripts/sendEventReminderEmails.py]: currentTerm here doesn't have all the data in it which is risky if EmailHandler needed something other than id.

File: app/scripts/sendEventReminderEmails.py [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+                            "recipientsCategory":"Interested",
+                            "subject":templateSubject,
+                            "body":templateBody}
+            sendEmail = EmailHandler(emailData, "172.31.3.239:8080", User.get_by_id("ramsayb2"))
+            sendEmail.send_email()
+            counter+=1
+        return counter
+    except (TypeError) as e:
+        print("No events were found.")
+        return 0
```
Comment (BrianRamsay) [inline app/scripts/sendEventReminderEmails.py:11]: I'm skeptical that this is necessary

Comment (BrianRamsay) [reply app/scripts/sendEventReminderEmails.py:11]: And if it is necessary, we should figure out why

**Review by BrianRamsay — CHANGES_REQUESTED** [review]

File: app/logic/events.py (hunk 1 above)
Comment (bledsoef) [reply app/logic/events.py:327]: Nice try Zach.

Comment (zachneill) [reply app/logic/events.py:327]: _Lame_

**Review by BrianRamsay — APPROVED** [review]
Tada!

**Pull request merged**